app/auth.go

```diff
@@ -208,6 +214,40 @@ func NewAuthManager(conf AuthManagerConfig) *MenderAuthManager {
 	return mgr
 }
 
+// handleCachedTenantToken Handle the cached tenant token if it is not the same
+// as the one in the configuration
+func handleCachedAuthorizationToken(db store.Store, serverURL, tenantToken []byte) error {
+	dbValue := append(serverURL, tenantToken...)
```

Comment:
Concatenating and using both as a value, instead of storing both. Tell me if you disagree with this approach

Reply:
I think that's a good approach, but I suggest something to separate them, like `___` or something.

Also, I would change the name of the database key then. Maybe this?

```
AuthTokenCacheInvalidatorName = "auth-token-cache-invalidator"
```